src/components/dashboard/Settings.js

```diff
@@ -199,6 +278,68 @@ const Settings = ({ screenProps, styles, theme, navigation }) => {
               />
             ))}
           </Section.Stack>
+
+          <Section.Row justifyContent="center" style={[styles.subtitleRow, { flexDirection: 'column' }]}>
+            <Section.Text fontWeight="bold" color="gray">
+              {t`Language`}
+            </Section.Text>
+            <Section.Stack justifyContent="flex-start" style={styles.selectLanguageContainer}>
+              <Section.Row style={styles.languageRow}>
+                <View style={styles.languageInputContainer}>
+                  <ModalDropdown
+                    defaultValue={languageCustomLabels[countryCode] ?? 'Select a language...'}
+                    options={supportedCountryCodes}
```

Comment:
```suggestion
                    options={[null, ...supportedCountryCodes]}
```

Reply:
The suggestions would not work.
1. option cannot be null because of string methods called in dropdownmodal
2. if language is null, it doesn't go through the process of trying to load a new locale: https://github.com/GoodDollar/GoodDAPP/blob/2290cd0d731cd06a314008577182bc25d0bd8bd0/src/language/i18n.js#L143

the dynamicActivate is where storage is updated and the restart/reload happens

Reply:
> The suggestions would not work.
> 
> 1. option cannot be null because of string methods called in dropdownmodal
> 2. if language is null, it doesn't go through the process of trying to load a new locale: https://github.com/GoodDollar/GoodDAPP/blob/2290cd0d731cd06a314008577182bc25d0bd8bd0/src/language/i18n.js#L143
> 
> the dynamicActivate is where storage is updated and the restart/reload happens

a few lines below:

```
useEffect(() => {
    I18n.getInitialLocale().then(setLanguage)
  }, [setLanguage])
```

Reply:
Regarding 1. - it could be an empty string too

Reply:
Mm have to retest then.
Maybe the flow/useffect of initialLocale was/is not triggered correctly